pkgs/tools/networking/quickserve/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchzip, python3, python3Packages, writeScript }: let
+  threaded_servers = python3Packages.buildPythonPackage {
```

Comment:
I think it would be more clean if the threaded_server module is included into `pkgs/python-modules` instead of inlining it here.

Reply:
IMHO, this would only make sense if it would ever be used by something else than quickserve.
But now threaded_server is basically quickserve.